frontend/lint/eslint-rules/no-color-literals.js

```diff
@@ -8,7 +8,7 @@
 //------------------------------------------------------------------------------
 
 const COLOR_REGEX =
-  /(?:#[a-fA-F0-9]{3}(?:[a-fA-F0-9]{3})?\b|(?:rgb|hsl)a?\(\s*\d+\s*(?:,\s*\d+(?:\.\d+)?%?\s*){2,3}\))/g;
+  /(?:#[a-fA-F0-9]{3}(?:[a-fA-F0-9]{3})?\b|(?:rgb|hsl)a?\(\s*\d+\s*(?:,\s*\d+(?:\.\d+)?%?\s*){2,3}\))/;
```

Comment:
The `g` flag would prevent all violations from being detected.

[before](https://github.com/user-attachments/assets/c2a493f4-aa1a-4664-bf3b-bfb7c538a1a8) vs [after](https://github.com/user-attachments/assets/6b22c377-9539-4d28-b180-8fbe27986b1e)